liquibase-core/src/main/java/liquibase/command/core/HistoryCommandStep.java

```diff
@@ -37,6 +38,10 @@ public class HistoryCommandStep extends AbstractCliWrapperCommandStep {
                 .build();
         CHANGELOG_FILE_ARG = builder.argument(CommonArgumentNames.CHANGELOG_FILE, String.class)
                 .description("The root changelog").build();
+        FORMAT_ARG = builder.argument("format", HistoryFormat.class)
```

Comment:
I think after fixing the current failing tests we can try setting up `TABULAR` as the default value for this configuration, as this is going to provide a more readable output. We can see if there are no failing tests or whether it's too time-consuming to fix them.

Reply:
Fixed in f6ed6dc3d314e65c67bc1c9a66a3b585c4603463